.changeset/rare-buses-swim.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/backend-tasks': major
```

Comment:
Not major though! Just minor. Also remember the DCO signature whenever you make commits.